velox/functions/sparksql/Arithmetic.h

```diff
@@ -212,4 +212,16 @@ struct SinhFunction {
   }
 };
 
+template <typename T>
+struct Log1pFunction {
+  FOLLY_ALWAYS_INLINE bool call(double& result, double a) {
+    double xAsymptote = -1.0;
```

Comment:
this variable is not needed; let's remove.

in the future, use static const double kXxx; for constants

Reply:
Removed xAsymptote variable.